azkaban-common/src/test/java/azkaban/executor/SelectorTest.java

```diff
@@ -78,7 +78,7 @@ public int compareTo(MockExecutorObject o) {
   }
 
   // Mock flow object.
-  protected class MockFlowObject{
+  protected static class MockFlowObject{
```

Comment:
`protected static` seems a bit confusing.
http://stackoverflow.com/questions/24289070/why-we-should-not-use-protected-static-in-java

Should we just make it package private or public (if it is accessed externally)?
There are other similar changes in this PR.